pontoon/insights/templates/insights/widgets/insights.html

```diff
@@ -48,63 +47,59 @@ <h3 class="controls">
         </section>
         {% endif %}
         {% if unreviewed_lifespans %}
-        <section class="half">
+        <section class="half suggestions-age">
             <div class="block clearfix">
-                <h3 class="controls">
-                    Age of unreviewed suggestions
-                    {{ Tooltip.display(
-                        intro='How old are the unreviewed suggestions on average at a particular point in time.',
-                    ) }}
-                </h3>
-                <canvas
-                    id="unreviewed-suggestions-lifespan-chart"
-                    data-lifespans="{{ unreviewed_lifespans }}"
-                    width="410"
-                    height="160">
-                </canvas>
+                <div class="suggestions-age-items">
+                    <div class="suggestions-age-item time-to-review-suggestions">
+                        <h3 class="controls">
+                            Time to review suggestions
+                            {{ Tooltip.display(
+                                intro='How much time it takes on average to review a suggestion.',
+                                items=[{
+                                    'class': 'within-a-month',
+                                    'name': 'Within a month',
+                                    'definition': 'Average age of suggestions reviewed during each month.',
+                                }, {
+                                    'class': 'twelve-month-average',
+                                    'name': '12-month average',
+                                    'definition': 'Average age of suggestions reviewed in the 12 months before each month.',
```

Comment:
```suggestion
                                    'definition': 'Average age of suggestions reviewed in the 12 months before the selected month.',
```

Not sure if `selected` is the best choice, but `hovered` might be more confusing than helping. For this one, I'm fine also to keep as-is.

Reply:
The spec says "specific", shall we stick to that? Not sure why I changed it in first place...